concord_cli/deploy.py

```diff
@@ -107,14 +110,9 @@ def parseFile(filename, parser):
     return conf
 
 def generate_options():
```

Comment:
there is only one argument. The zk_path and zk_hosts are taken from the manifest file anyway